dynamiqs/integrators/apis/dssesolve.py

```diff
@@ -28,18 +28,21 @@ def dssesolve(
         This function has not been implemented yet. The following API is indicative
         of the planned implementation.
 
+    In quantum optics, the diffusive SSE describes the evolution of a quantum
+    system under homodyne or heterodyne detection with unit detection efficiency.
     This function computes the evolution of the state vector $\ket{\psi(t)}$ at time $t$,
     starting from an initial state $\ket{\psi_0}$ according to the diffusive SSE in Itô
-    form (with $\hbar=1$ and where time is implicit(1))
+    form ($\hbar=1$, time is implicit(1))
```

Comment:
Why not, I propose to do this in another PR for all solvers.

Reply:
Ok, I prefer this succint formulation on my end.